rholang/examples/bond/bond.rho

```diff
@@ -0,0 +1,8 @@
+new retCh, PoSCh, rl(`rho:registry:lookup`), stdout(`rho:io:stdout`) in {
+  stdout!("About to lookup pos contract") |
+  rl!(`rho:rchain:pos`, *PoSCh) |
+  for(@(_, PoS) <- PoSCh) {
+    stdout!("About to bond") |
+    @PoS!("bond", 100, *retCh)
```

Comment:
One more thing to point out during the demo is that this contract is going to accept a `deployerId` in the future. In it's current form, this relies on the (generalized) 'drain vault' vulnerability

Reply:
Noted